src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -151,6 +151,9 @@ const WILDCARD_FITLER = {
 }
 
 const getCql = ({ filterTree, cql }) => {
+  if (filterTree.filters && filterTree.filters.length == 0) {
```

Comment:
This could result in an error if filterTree was undefined. Could use getIn(filterTree, ['filters','length'], 0) and check to see that it's not 0.

Reply:
We should also be using `===` just to be safe

Reply:
The problem I was seeing is that `getCql` also gets hit when loading up the worskpaces the filterTree was in a state for `getIn(filterTree, ['filters','length'], 0) ` to be true which we dont want and  thought of an additional check if an actual filter exist rather than bieng undefined . This is whats currently working 
|  Component  | getIn(filterTree, ['filters', 'length'], 0) == 0          |  Array.isArray(getIn(filterTree, ['filters'], 0))       |
| ------------- |:-------------:|:-----:|
| Workspace load     | true | false |
| Metacard click     |  false   |  true |
| Empty Advanced Search | true |  true |


```javascript  
const checkForEmptySearch =
    Array.isArray(getIn(filterTree, ['filters'], 0)) &&
    getIn(filterTree, ['filters', 'length'], 0) == 0
```